tgui/packages/tgui/interfaces/BankingComputer/CrewAccounts.tsx

```diff
@@ -170,32 +173,36 @@ export const CrewAccounts = (props: CrewAccountsProps, context) => {
 
   };
 
-  const searchAndSortCrewAccounts = (filter: string, text: string, sortConfig: CrewColumnSortConfig) => {
+  const searchAndSortCrewAccounts = (
+    accounts: CrewAccount[],
+    filter: string,
+    text: string,
+    sortConfig: CrewColumnSortConfig) => {
 
-    let accounts = props.accounts;
+    let accountsToSort = accounts;
     if (sortConfig) {
-      accounts.sort((a, b) => compareByField(a, b, sortConfig));
+      accountsToSort = [...accounts].sort((a, b) => compareByField(a, b, sortConfig));
 
       if (sortConfig.dir === SortDirection.Asc) {
-        accounts.reverse();
+        accountsToSort.reverse();
```

Comment:
Not too opposed to this, but would prefer the `.sort` comparator could be configured with a direction. This would reduce the amount of operations on the data.